src/chunk_type.rs

```diff
@@ -1,148 +1,96 @@
-use crate::error::PngMeError;
-use std::convert::TryFrom;
-use std::fmt;
+use core::fmt;
 use std::str::FromStr;
 
-/// Byte representation of a PNG version 1.2 Chunk.
-/// `ChunkType` provides validation for a PNG version 1.2
-/// chunk, also provides construction and conversion for the
-/// same type.
-///
-/// # Structure
-///
-/// ```ignore
-/// bLOb  <-- 32 bit chunk type code represented in text form
-/// ||||
-/// |||+- Safe-to-copy bit is 1 (lowercase letter; bit 5 is 1)
-/// ||+-- Reserved bit is 0     (uppercase letter; bit 5 is 0)
-/// |+--- Private bit is 0      (uppercase letter; bit 5 is 0)
-/// +---- Ancillary bit is 1    (lowercase letter; bit 5 is 1)
-///```
-/// [Source](http://www.libpng.org/pub/png/spec/1.2/PNG-Structure.html)
-///
```

Comment:
I would like to keep these docs as they are insightful. Can we have them in the module scope perhaps?